app/Http/Controllers/Dormitory/PrintController.php

```diff
@@ -36,6 +38,16 @@ public function index() {
                 "free_pages" => Auth::user()->sumOfActiveFreePages()
             ]);
     }
+    public function noPaper(){
+        $userName = Auth::user()->name;
+        $users = DB::table('roles')->select('email')
```

Comment:
You should be using [Eloquent models](https://laravel.com/docs/8.x/eloquent).
It would look something like this:
```
$users = User::role(Role::INTERNET_ADMIN)->get();
```

Reply:
Actually, you can pass the user to Mail::to, which is better since then we can pick up this user's locale preference, etc. I changed this as well.